test/approximate-number_test.js

```diff
@@ -356,4 +356,60 @@ describe('approximate-number', function() {
       });
     });
   });
+
+  describe('options.precision > 0', function() {
+    var options = {
+      precision: 3
+    };
+
+    // input => output
+    var tests = {
+      0: '0',
+      0.1: '0.1',
+      1: '1',
+      10: '10',
+      999: '999',
+      1000: '1k',
+      1001: '1k',
+      1234: '1.23k',
```

Comment:
Some of these don't seem like what I would expect - if precision is `3`, then shouldn't this be `1.234k`?

Reply:
Oh, wait, I was confusing `toFixed()` with `toPrecision()`

Reply:
It's intended to be the number of significant digits, not the number of decimal places. So you should see 1.23, 12.3, 123, 1230, etc.